signac_dashboard/dashboard.py

```diff
@@ -373,6 +373,21 @@ def _job_search(self, query):
                 try:
                     f = json.loads(query)
                 except json.JSONDecodeError:
+                    if "True" in query and "False" in query:
+                        flash(
+                            'Interpreting "True" and "False" as strings. For boolean values use "true" and "false".',  # noqa:E501
```

Comment:
```suggestion
                            'Interpreting "True" and "False" as strings. For'
							'boolean values use "true" and "false".',
```
As Bradley suggested, wrap the string appropriately instead of ignoring the flake8 error.